src/lib/Behat/Page/ContentUpdateItemPage.php

```diff
@@ -170,4 +172,20 @@ public function verifyFieldCannotBeEditedDueToLimitation(string $fieldName)
         $fieldLocator = new VisibleCSSLocator('', sprintf($this->getLocator('activeNthField')->getSelector(), $this->getFieldPosition($fieldName)));
         $this->getHTMLPage()->find($fieldLocator)->assert()->hasClass('ez-field-edit--disabled');
     }
+
+    public function verifyAutosaveNotificationIsDisplayed(): void
+    {
+        Assert::assertContains(
+            'Autosave is on.',
+            $this->getHTMLPage()->find($this->getLocator('autosaveInfo'))->getText()
```

Comment:
Please use `$this->getHTMLPage()->find($this->getLocator('autosaveInfo'))->assert()->textContains(...)` instead